### PR TITLE
Release 5.4.4

### PR DESCRIPTION
**Pull request by bojankoma**
* Compile with SDK 32.
* Update dependencies to stable version.
* Switch to Material3 theme.
* Replace FuzzySearch library with Levenshtein algorithm implementation.

### CODE REVIEW

File: dbinspector/src/main/kotlin/com/infinum/dbinspector/data/sources/memory/distance/LevenshteinDistance.kt [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+ *
+ * It is the number of changes needed to change one String into another,
+ * where each change is a single character modification.
+ *
+ */
+internal class LevenshteinDistance : Sources.Memory.Distance {
+
+    /**
+     * Iterates over options and calculates unlimited distance between each option and query String.
+     * Maps a tuble of index, optiona and distance, then finds minimum distance tuple and returns index.
```
Comment (JonatanPlesko) [inline dbinspector/src/main/kotlin/com/infinum/dbinspector/data/sources/memory/distance/LevenshteinDistance.kt:17]:
```suggestion
     * Maps a tuple of index, option, and distance, then finds minimum distance tuple and returns index.
```

File: dbinspector/src/main/kotlin/com/infinum/dbinspector/data/sources/memory/distance/LevenshteinDistance.kt [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        }
+            .minByOrNull { it.third }
+            ?.first
+
+    /**
+     * Calculates Levenshtein distance between two Strings without a threshold to pass.
+     * A higher score indicates a greater distance.
+     *
+     * calculateUnlimited("scar", "car")       = 1
+     * calculateUnlimited("car", "mug")        = 3
```
Comment (JonatanPlesko) [inline dbinspector/src/main/kotlin/com/infinum/dbinspector/data/sources/memory/distance/LevenshteinDistance.kt:33]: Nice examples 👍

File: gradle.properties [hunk 3]
```diff
@@ -1,21 +1,5 @@
...
 
-# When configured, Gradle will run in incubating parallel mode.
-# This option should only be used with decoupled projects. More details, visit
-# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
-
 android.useAndroidX=true
-android.enableJetifier=false
\ No newline at end of file
+android.enableJetifier=false
```
Comment (JonatanPlesko) [inline gradle.properties:4]: This is false by default, maybe we can simply remove it.

Also, have you thought about turning on the [build cache](https://docs.gradle.org/current/userguide/build_cache.html)?

**Review by JonatanPlesko — APPROVED** [review]
Looks good 👍

File: gradle.properties (hunk 3 above)
Comment (bojankoma) [reply gradle.properties:4]: I'll consider it both. But not a concern for this PR.

**Pull request merged**